apps/condo/lang/en/en.json

```diff
@@ -1938,7 +1939,7 @@
   "tour.pageData.billing.completed.title": "Have you worked on reducing debit debt",
   "tour.pageData.billing.completed.subtitle": "Now residents can pay the receipts, and you – follow the receipts",
   "tour.pageData.ticket.todo.title": "Optimize tickets management",
-  "tour.pageData.ticket.todo.subtitle": "When you complete all of the tasks in the list, residents will be able to submit tickets through the mobile app, and you will see them on the platform.",
+  "tour.pageData.ticket.todo.subtitle": "When you set up work with tickets, you will be able to process them faster. For example, communicate in chats about a specific tickets. Technic App{andLinkToEmployeeBot} will help you keep abreast of changes. And residents will have the opportunity to evaluate tickets.",
```

Comment:
Seems like a " " is missing `Technic App{andLinkToEmployeeBot}` -> `Technic App {andLinkToEmployeeBot}`

Reply:
This was intended so that there would be no double space.  You can watch preview